shenyu-client/shenyu-client-dubbo/shenyu-client-alibaba-dubbo/src/main/java/org/apache/shenyu/client/alibaba/dubbo/annotation/ShenyuService.java

```diff
@@ -0,0 +1,434 @@
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.client.alibaba.dubbo.annotation;
+
+import com.alibaba.dubbo.config.annotation.Method;
+import com.alibaba.dubbo.config.annotation.Service;
+import org.apache.shenyu.client.dubbo.common.annotation.ShenyuDubboClient;
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.stereotype.Component;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * The interface shenyu service.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Service
+@Inherited
+@ShenyuDubboClient
+public @interface ShenyuService {
+
+    /**
+     * Path string.
+     *
+     * @return the string
+     */
+    @AliasFor(annotation = Component.class)
+    String value() default "";
+
+    /**
+     * Alias for {@link ShenyuDubboClient#ruleName()} .
+     *
+     * @return the string
+     */
+    @AliasFor(annotation = ShenyuDubboClient.class)
+    String ruleName() default "";
+
+    /**
+     * Alias for {@link ShenyuDubboClient#desc()} .
+     *
+     * @return String string
+     */
+    @AliasFor(annotation = ShenyuDubboClient.class)
+    String desc() default "";
+
+    /**
+     * Alias for {@link ShenyuDubboClient#enabled()} .
+     *
+     * @return the boolean
+     */
+    @AliasFor(annotation = ShenyuDubboClient.class)
+    boolean enabled() default true;
+
+    /**
+     * Alias for {@link Service#interfaceClass()}.
+     *
+     * @return the string
+     */
+    @AliasFor(annotation = Service.class)
+    Class<?> interfaceClass() default void.class;
+
+    /**
+     * Alias for {@link Service#interfaceName()} .
+     *
+     * @return the string
+     */
+    @AliasFor(annotation = Service.class)
+    String interfaceName() default "";
+
+    /**
+     * Alias for {@link Service#version()} .
+     *
+     * @return the string
+     */
+    @AliasFor(annotation = Service.class)
+    String version() default "";
+
+    /**
+     * Alias for {@link Service#group()} .
+     *
+     * @return the string
+     */
+    @AliasFor(annotation = Service.class)
+    String group() default "";
+
+    /**
+     * Alias for {@link Service#path()}.
+     *
+     * @return the string
+     */
+    @AliasFor(annotation = Service.class)
+    String path() default "";
```

Comment:
change to `@AliasFor(annotation = ShenyuDubboClient.class)` and add a `dubboPath` method.

Reply:
Done.